graylog2-web-interface/src/views/components/visualizations/bar/bindings.tsx

```diff
@@ -25,15 +25,18 @@ type BarVisualizationConfigFormValues = {
   barmode: 'group' | 'stack' | 'relative' | 'overlay',
 };
 
+const DEFAULT_BARMODE = 'group';
+
 const validate = hasAtLeastOneMetric('Bar chart');
 
-const barChart: VisualizationType = {
+const barChart: VisualizationType<BarVisualizationConfig, BarVisualizationConfigFormValues> = {
   type: BarVisualization.type,
   displayName: 'Bar Chart',
   component: BarVisualization,
   config: {
-    fromConfig: (config: BarVisualizationConfig | undefined): BarVisualizationConfigFormValues => ({ barmode: config?.barmode ?? 'group' }),
-    toConfig: (formValues: BarVisualizationConfigFormValues): BarVisualizationConfig => BarVisualizationConfig.create(formValues.barmode),
+    createConfig: () => ({ barmode: DEFAULT_BARMODE }),
+    fromConfig: (config: BarVisualizationConfig | undefined) => ({ barmode: config?.barmode ?? 'group' }),
```

Comment:
Let's use the `DEFAULT_BARMODE` for the `barmode` fallback as well.

Reply:
:+1: